{{copier__project_slug}}/k8s/sandbox/postgres.cnpg.yaml

```
@@ -6,9 +6,43 @@ metadata:
     app: postgres
 spec:
   instances: 1
-  envFrom:
-  - configMapRef:
-      name: app-config
+  bootstrap:
+    initdb:
+      database: POSTGRES_DB
+      owner: POSTGRES_USER
   primaryUpdateStrategy: unsupervised
   storage:
     size: 2Gi
+
+  backup:
+    barmanObjectStore:
+      destinationPath: AWS_S3_BACKUP_KUSTOMIZED
+      s3Credentials:
+        accessKeyId:
+          name: aws-creds
+          key: AWS_S3_ACCESS_KEY_ID
+        secretAccessKey:
+          name: aws-creds
+          key: AWS_S3_SECRET_ACCESS_KEY
+      wal:
+        compression: gzip
+      data:
+        compression: gzip
+        jobs: 1
+      tags:
+        backupRetentionPolicy: "expire"
+      historyTags:
+        backupRetentionPolicy: "keep"
+    retentionPolicy: "30d"
+---
+apiVersion: postgresql.cnpg.io/v1
+kind: ScheduledBackup
+metadata:
+  name: scheduled-backup
+spec:
+  schedule: "0 0 * * 7"  # Runs weekly at midnight on Sunday
+  suspend: false
+  immediate: true
+  backupOwnerReference: self
+  cluster:
+    name: postgres
```

We need to backup daily:
```suggestion
apiVersion: postgresql.cnpg.io/v1
kind: ScheduledBackup
metadata:
  name: postgres-daily
spec:
  schedule: "0 0 0 * * *"
  backupOwnerReference: cluster
  cluster:
    name: postgres
```